app/views/EntryEdit/LeftPane/SimplifiedTextView/styles.css

```diff
@@ -12,6 +12,28 @@
         user-select: none;
     }
 
+    .simplified-text {
+        line-height: 1.5;
+        --base-font-size: var(--dui-font-size-medium);
+        --diff: 2px;
+
+        &.extra-small{
+            font-size: calc(var(--base-font-size) - 2 * var(--diff));
+        }
```

Comment:
I think the font we have should be the smallest allowed font and only allow increments. What do you think?

Reply:
That is what the process is now. Take a look please.